api/src/about/dto/about.dto.ts

```diff
@@ -0,0 +1,15 @@
+export class AboutDto {
```

Comment:
I would reconsider this model.

* there shouldn't be a model called progress bar, it is not descriptive nor concise at all (progress bar for what?), you would rather create separate resource/model with a one to many relation and I would call it `skills` 
* then you should start thinking about what a skill should look like from a database point of view, which fields are important for one skill, which fields should I consider for data modeling and which should be left out for presentation details (like color and icon, these can be handled by the frontend)
* in the skill model, you may consider renaming ~`progress`~ property to `level` and it should be a number between 1 and 100 for example

Reply:
changes in consideration